src/main/java/www/purple/mixxy/models/User.java

```diff
@@ -1,26 +1,54 @@
 package www.purple.mixxy.models;
 
+/**
+ * Modified by Brian_Sabz on 4/5/16.
+ * 
+ * @author Brian_Sabz
+ */
+
 import com.googlecode.objectify.annotation.Entity;
 import com.googlecode.objectify.annotation.Id;
 import com.googlecode.objectify.annotation.Index;
 
+import java.util.Date;
+
 @Entity
 @Index
 public class User {
-    
-    @Id
-    public Long id;
-    public String username;
-    public String password;
-    public String fullname;
-    public boolean isAdmin;
-
-    public User() { /* Needed by Objectify */ }
-    
-    public User(final String username, final String password, final String fullname) {
-        this.username = username;
-        this.password = password;
-        this.fullname = fullname;
-    }
- 
+
+	@Id
+	public Long id;
+	public String username;
+	public String firstname;
+	public String lastname;
+	public String email;
+
+	// @Chris - TODO: how are we going to use this map?
+	// public Map<OAuthService, String> authentications = new HashMap<>();
+
+	public Role role;
```

Comment:
We should do a [partial index](https://github.com/objectify/objectify/wiki/Queries#partial-indexes) on this so we can quickly find admins (and maybe draw an icon next to their names or something).  You often don't care about finding every *user*, but you might for finding every *staff* member.